docs/Architecture.md

```diff
@@ -15,6 +15,7 @@ Pelorus is composed of the following open source components.
     * Commit Time
     * Deploy Time
     * Failure
+    * Webhook Exporter
```

Comment:
just Webhook? this bullet is already for exporters

Reply:
changed.